### PR TITLE
chore: release v2.0.0

### PR DESCRIPTION
**Pull request by eunjae-lee**
This pull request prepares the following release:
| Repository | Branch | Update | Change |
|---|---|---|---|
| [search-insights.js](https://github.com/algolia/search-insights.js) | releases/v2.0.0 | major | [`1.8.0` → `2.0.0`](https://github.com/algolia/search-insights.js/compare/v1.8.0...releases/v2.0.0) |

### Release Summary
This is going to be published with the following command:

```npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest```

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release v2.0.0`.
<details>
<summary>See details</summary>

After that, a commit `chore: release v2.0.0` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# [2.0.0](https://github.com/algolia/search-insights.js/compare/v1.8.0...v2.0.0) (2021-04-28)
+
+
+### BREAKING CHANGES
+
+* **cookie:** It used to be `true`, but to be GDPR compliant, we're changing it to `false` by default.
```
Comment (Haroenv) [inline CHANGELOG.md:6]:
```suggestion
* **useCookie:** It used to be `true`, but to be GDPR compliant, we're changing it to `false` by default.
```

**Comment by Haroenv** [comment]
do we consider this for v2 or remove the comment? cc @tkrugg, you wrote it originally

https://github.com/algolia/search-insights.js/issues/127

**Comment by eunjae-lee** [comment]
I think it's fixed by https://github.com/algolia/search-insights.js/pull/128

**Comment by Haroenv** [comment]
I linked the wrong code, in #128 it's mentioned it can be better after a breaking change: 

https://github.com/algolia/search-insights.js/blob/c47b405fddb3175048ad631e6aa115bc88865520/lib/_processQueue.ts#L30-L37

**Comment by eunjae-lee** [comment]
↑ what do you think @tkrugg ?

**Comment by eunjae-lee** [comment]
@Haroenv 

> Reassigning the pointer is a bad idea

Not sure why it's a bad idea. Reassigning the reference of window.aa is bad, though.

> for next major version, use a custom method instead of push.

If we use a custom method, then we're going to have to change the snippet

```html
<script>
var ALGOLIA_INSIGHTS_SRC = "https://cdn.jsdelivr.net/npm/search-insights@1.8.0";

!function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e[s]=e[s]||function(){
(e[s].queue=e[s].queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],
i.async=1,i.src=n,c.parentNode.insertBefore(i,c)
}(window,document,"script",ALGOLIA_INSIGHTS_SRC,"aa");
</script>
```

From
`(e[s].queue=e[s].queue||[]).push(arguments)`
to something like
`(e[s].queue=e[s].queue||[]).add(arguments)`

It's a major update, so we can have a big banner about the snippet change, but I don't see why we should use a custom method.

What do you think?

**Review by Haroenv — APPROVED** [review]
let's go with this version then!

**Pull request merged**